.travis.yml

```diff
@@ -26,12 +26,14 @@ install:
   # Install dependencies
   - conda create -n test-environment python=$PYTHON
   - source activate test-environment
-  - conda install -c conda-forge numpy=$NUMPY scipy pytables h5py bcolz pytest coverage toolz scikit-learn cytoolz chest blosc cython psutil ipython
-  - if [[ $PYTHON != '3.3' ]]; then conda install -c conda-forge pandas=$PANDAS distributed cloudpickle bokeh; fi
+  - conda install -c conda-forge numpy=$NUMPY scipy pytables h5py bcolz pytest coverage toolz scikit-learn cytoolz chest blosc cython psutil ipython python-lmdb
+  - if [[ $PYTHON != '3.3' ]]; then conda install -c conda-forge pandas=$PANDAS cloudpickle bokeh; fi
   - if [[ $PYTHON == '3.3' ]]; then pip install cloudpickle pandas==$PANDAS; fi
   - if [[ $PYTHON == '2.7' ]]; then pip install backports.lzma mock; fi
   - pip install git+https://github.com/mrocklin/partd --upgrade
   - pip install git+https://github.com/mrocklin/cachey --upgrade
+  - pip install git+https://github.com/dask/zict --upgrade
+  - pip install git+https://github.com/dask/distributed --upgrade
```

Comment:
s3fs is included as a dependency in the `distributed` conda package but not in the PyPI package.  We keep the PyPI packages lightweight.